src/embed.ts

```diff
@@ -214,6 +214,7 @@ export default async function embed(
         .attr('href', '#')
         .on('click', () => {
           post(window, editorUrl, {
+            config: config ? JSON.stringify(config, null, 2) : null,
```

Comment:
Config shouldn't be a string. We expect the spec to be a string because we want to preserve formatting. It's not relevant for a config.

Reply:
Okay then, will send it without stringifying.

Reply:
👍